Security/day03/Easy/chall-01-Lenna/Makefile

```diff
@@ -0,0 +1,31 @@
+PREFIX ?= pathwar/
```

Comment:
useless file ?

Reply:
delete all challenges and just give a readme to give the the url of pathwar